hack/jenkins/e2e.sh

```diff
@@ -485,6 +487,27 @@ case ${JOB_NAME} in
     : ${ENABLE_DEPLOYMENTS:=true}
     ;;
 
+  # Runs "disruptive" tests on GCE on the release candidate branch,
+  # sequentially, against the latest 1.1 ci release.
+  kubernetes-e2e-gce-disruptive-1.1)
+    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-disruptive-1.1"}
+    : ${E2E_DOWN:="false"}
+    : ${E2E_NETWORK:="gce-e2e-disruptive-1-1"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
+          ${REBOOT_SKIP_TESTS[@]:+${REBOOT_SKIP_TESTS[@]}}\
+          ) --ginkgo.focus=$(join_regex_no_empty \
+          ${DISRUPTIVE_TESTS[@]:+${DISRUPTIVE_TESTS[@]}} \
+          "GCE\sL7\sLoadBalancer\sController"
+          )"}
+    : ${JENKINS_PUBLISHED_VERSION:="ci/latest-1.1"}
+    : ${KUBE_GCE_INSTANCE_PREFIX="e2e-gce-disruptive-1-1"}
+    : ${KUBE_GCS_STAGING_PATH_SUFFIX:="disruptive-1.1"}
+    # TODO: move into its own project
+    : ${PROJECT:="kubernetes-jenkins"}
```

Comment:
Fair question... I copied this from ixdy@'s fork which the test is currently running from. So probably, but we do want to move it.